packages/react-components/react-button/library/src/components/SplitButton/useSplitButtonStyles.styles.ts

```diff
@@ -169,20 +172,53 @@ const useRootStyles = makeStyles({
       },
     },
   },
+
+  largeChevron: {
+    [`& .${menuButtonIconClass}`]: {
+      fontSize: tokens.fontSizeBase400,
+      width: tokens.fontSizeBase400,
+      height: tokens.fontSizeBase400,
+    },
+  },
+});
+
+const useIconOnlyStyles = makeStyles({
+  outline: {},
+  secondary: {},
+  primary: {},
+  subtle: {},
+  transparent: {
+    [`& .${splitButtonClassNames.primaryActionButton}`]: {
+      paddingRight: tokens.spacingVerticalNone,
+    },
+    [`& .${splitButtonClassNames.menuButton}`]: {
+      paddingLeft: tokens.spacingVerticalNone,
+      paddingRight: tokens.spacingVerticalNone,
+    },
```

Comment:
I just noticed that you have access to these slots directly, so you should probably apply these styles directly and not through a children selector to not mess with style specification.